manual/content/point-data/point-data.md

```diff
@@ -0,0 +1,349 @@
+
+        -- Per default, the chosen parameter is set to the first one in the list above.
+        -- However, you can also set the default parameter in the asset
+        Parameter = "name of parameter",
+        -- It is also possible to set a value range that is different compared to the
+        -- one for the parameter option. This will be used at startup, but overwritten
+        -- if the parameter option is changed
+        ValueRange = { 5.0, 10.0 }
+    }
+  },
+  ...
+```
+
+:::{figure} colormap_ui_annotated.png
+:align: center
+*The chosen set of parameters will be available in the user interface. Here we see the chosen parameter (A), which can be changed through a drop-down menu, and the currently selected value range (B). The value range (B) will automatically update when a parameter (A) is chosen, but it is also possible to set it explicitly. There is also a button for setting the range to the min and max value in the dataset (C).*
+:::
+
+#### Missing values (NaN) and Values outside the range
+
+Finally, it is also possible to color points outside the specified value range differently compared to the ones within the range, as well as showing missing data points in a specific color.
+
+We refer to our example assets for more details on how to customize these color map settings, but here follows a summary of the properties for controlling what happens outside the data range or for missing values. All of these are optional but can be added to the `ColorMapping` table in the asset.
+
+| Properties| Description | Type |
+| :--- | :--- | :--- |
+| `NoDataColor`*, `ShowMissingData` | Show missing data points in a specific color | `vec4`, `bool` |
+| `HideValuesOutsideRange` | Hide any value that is outside the provided value range | `bool` |
+| `AboveRangeColor`*, `UseAboveRangeColor` | Show points with values larger than the max value in the range in a specific color | `vec4`, `bool` |
+| `BelowRangeColor`*, `UseBelowRangeColor` | Show points with values below than the min value range in a specific color | `vec4`, `bool` |
+
+\* Note that these colors can also be set in the color map file itself. See [the page on color map files](./data-formats.md#color-maps-cmap) for more details.
+
+## Adding a Texture
+
+A sprite texture (i.e. an image) can be used to decide the shape of the points. To add a texture, simply provide a path to the image you want to use in the asset file:
+
+```lua
+  ...
+  Renderable = {
+    Type = "RenderablePointCloud",
+    File = asset.resource("path/to/dataset.csv"),
+    -- Add a texture to the points (a variety of image formats are supported, not only .png)
+    Texture = asset.resource("path/to/texture.png")
+  },
+  ...
+```
+
+The points will look the best with textures that have a transparent background. Below is an example of a point cloud with a star shape. Also, if you want to apply a color to your point using a color map or fixed color, we recommend using texture in white or bright grayscale.
```

Comment:
Recommend a simple mention of transparent in this case being equal to zero alpha, since you discuss alpha channel in the data-formats page. Also mention transparency works even in the middle of an image, like the blue kepler rings.